src/Vault.sol

```diff
@@ -116,29 +118,32 @@ contract Vault is EIP1967Admin, VaultAccessControl, IERC721Receiver, ICDP, Multi
     /// @notice Mapping, returning set of all nfts, managed by vault
     mapping(uint256 => uint256[]) private _vaultNfts;
 
-    /// @notice Mapping, returning debt by vault id (in MUSD weis)
-    mapping(uint256 => uint256) public vaultDebt;
+    /// @notice Mapping, returning true if vault's NFT is exist and managed by this minter and false otherwise
+    mapping(uint256 => bool) private _isExists;
+
+    /// @notice Mapping, returning normalized debt by vault id (in MUSD weis)
+    mapping(uint256 => uint256) public vaultNormalizedDebt;
+
+    /// @notice Mapping, returning sum of all mints minus fraction of all repaid amounts
+    mapping(uint256 => uint256) public vaultMintsToBePaid;
 
     /// @notice Mapping, returning owed by vault id (in MUSD weis)
     mapping(uint256 => uint256) public vaultOwed;
 
-    /// @notice Mapping, returning total accumulated stabilising fees by vault id (which are due to be paid)
-    mapping(uint256 => uint256) public stabilisationFeeVaultSnapshot;
-
     /// @notice Mapping, returning id of a vault, that storing specific nft
     mapping(uint256 => uint256) public vaultIdByNft;
 
-    /// @notice Mapping, returning last cumulative sum of time-weighted debt fees by vault id, generated during last deposit / withdraw / mint / burn
-    mapping(uint256 => uint256) public globalStabilisationFeePerUSDVaultSnapshotD;
-
-    /// @notice State variable, returning current stabilisation fee (multiplied by DENOMINATOR)
-    uint256 public stabilisationFeeRateD;
+    /// @notice State variable, meaning total protocol debt on globalStabilisationFeePerUSDSnapshotTimestamp multiplied by DEBT_DENOMINATOR
+    uint256 public globalFeesRateDebtAccumulatorD;
```

Comment:
I think we can combine `globalFeesRateDebtAccumulatorD` and `globalFeesRateUpdateTimestamp` under a single storage slot. e.g. uint216 + uint40, since they are always read and modified together